styles

```diff
@@ -1 +1 @@
-Subproject commit 19dde3cb1c36fe5f42dafc78c806fd7aca98126c
+Subproject commit 014e36833420c81b29bec816b479ea61e94821d4
```

Comment:
Good catch @jfirebaugh!  We've gone through and cleaned up a few others as well.  Thanks!